src/sql/parts/modelComponents/components.contribution.ts

```diff
@@ -9,6 +9,7 @@ import CardComponent from './card.component';
 import InputBoxComponent from './inputbox.component';
 import DropDownComponent from './dropdown.component';
 import ButtonComponent from './button.component';
+import CheckBoxComponent from './checkbox.component';
```

Comment:
>CheckBoxComponent [](start = 7, length = 17)

@llali@microsoft.com Anthony already created angular version of checkbox, inputbox, and select box (C:\Repro\sqlopsstudio\src\sql\base\browser\ui\checkbox\checkbox.component.ts)

Reply:
@AbbiePetcht that's the one I'm using